eZ/Publish/Core/Repository/Values/Content/Content.php

```diff
@@ -57,6 +62,11 @@ public function __construct(array $data = [])
         }
     }
 
+    public function getThumbnail(): ?string
```

Comment:
Shouldn't this be added to `API\Values\Content` as well ?? I don't wanna typehint on `Core` :)